Dockerfile

```
@@ -1,5 +1,5 @@
 # Start with a rust alpine image
-FROM rust:1-alpine3.13
+FROM rust:1.57-alpine
```

```suggestion
FROM rust:1-alpine3.14
```
This should pin a specific alpine release, but pinning a rust compiler version is not necessary.